crates/bevy_math/src/cubic_splines.rs

```diff
@@ -1,17 +1,19 @@
 //! Provides types for building cubic splines for rendering curves and use with animation easing.
 
-use glam::{Vec2, Vec3, Vec3A};
-
 use std::{
     fmt::Debug,
     iter::Sum,
-    ops::{Add, Mul, Sub},
+    ops::{Add, Div, Mul, Sub},
 };
 
+use bevy_utils::{thiserror, thiserror::Error};
```

Comment:
You might consider pulling this dep in directly, so you don't need to build the entire utils crate first. I also recall this was a contentious thing when I first worked on this. Bevy math is very foundational, and sensitive to dependency count. Is there any reason we can't use rust's basic Error type?

Reply:
I initially used `thiserror` directly, but changed it from a suggestion in one of the reviews: https://github.com/bevyengine/bevy/pull/10701#discussion_r1468078528

Reply:
I don't think that is the right call here, that would triple bevy_math's dependency count.